packages/pagination/src/Pagination.js

```diff
@@ -25,8 +25,9 @@ const Pagination = ({
   onPageChange,
   children,
   watchList,
+  defaultPage,
 }) => {
-  const [currentPage, setPage] = useState(1);
+  const [currentPage, setPage] = useState(defaultPage || 1);
```

Comment:
what if `defaultPage` was explicitly set to `0` ? 

i'm thinking instead we add a default prop:

```
Pagination.defaultProps = {
    defaultPage: 1,
   ...
};
```

and then this line is:
```
const [currentPage, setPage] = useState(defaultPage);
```

Reply:
done!